### ISSUE TITLE
Replies in iOS

### ISSUE THREAD

**Issue by kintaro1981** [KnugiHK/WhatsApp-Chat-Exporter#28]
Hello, 

first of all thank you for your work! 
Now the request, iphone's "replies" are going to be supported in the near future or I don't have to old my breath?

**Comment by KnugiHK** [comment]
After #9 is completed and released, I will look into iPhone's reply.

**Comment by kintaro1981** [comment]
any news about this?

**Comment by KnugiHK** [comment]
> can I install the dev version with pip?

Yes. `pip install git+https://github.com/KnugiHK/Whatsapp-Chat-Exporter.git@dev`

**Comment by kintaro1981** [comment]
> I implemented a basic version of replying in iOS in [80c3ed1](https://github.com/KnugiHK/Whatsapp-Chat-Exporter/commit/80c3ed11f6f9df10df1000d549f2ada5f99e1106). Feel free to try it in the `dev` branch.

I tried and it is working. Thank you.

**Comment by KnugiHK** [comment]
It is not yet fully functional, as compared to Android's replies. Therefore, depending on the progress and needs, it may or may not include in the next release.

**Comment by KnugiHK** [comment]
It is included in the 0.9.5 release. However, I will keep this issue open.

**Comment by KnugiHK** [comment]
Implmented in #28.

**Comment by KnugiHK** [comment]
The message preview feature has been implemented in 510b4a7. This issue is now considered resolved.

**Comment by KnugiHK** [comment]
Released in 0.11.0.
